_entries/02-08 acr.md

```diff
@@ -74,6 +77,10 @@ After you grant your Kubernetes cluster access to your private registry, you can
 
 Kubernetes is declarative and keeps a manifest of all object resources. Edit your deployment object with the updated image.
 
+**Task Hints**
+* You can edit a live deployment object in Kubernetes with `kubectl edit deployment`. It is suggested to run `export KUBE_EDITOR="nano"` to stop `kubectl` from using the vi editor.
```

Comment:
What's wrong with vi? 😄

Reply:
😄 Nothing! Just many people running through this might not be comfortable with it! Trying to make the workshop as friendly as possible. And no one would call vi friendly 😆